social.md

```diff
@@ -76,7 +76,7 @@ The library must implement provider callbacks for all social provider
 directories that are mapped to the specified Stormpath application.  The
 callback URLs for each provider should take the form of:
 
-> `/<stormpath.web.socialProviders.callbackRoot>/<providerId>`
+> `<stormpath.web.socialProviders.[providerId].callbackUri>/<providerId>`
```

Comment:
I disagree with this - it should be `uri` because that is consistent with the rest of our entire framework config.  We have config representations that are shared by other endpoints and if you make this one a 'special case', now we have to parse it separately.

That a uri is used for callback is secondary to the fact that it is a `uri` within your application that is controlled by us (like any other `uri` property).  I don't want to create special parsing cases.